internal/tiltfile/tiltfile_test.go

```diff
@@ -2576,6 +2576,7 @@ default_registry("foo")
 docker_build('gcr.io/foo', 'foo')
 `)
 
+	f.loadErrString("Traceback ")
```

Comment:
doesn't matter that much, but each call to `loadErrString` will execute the Tiltfile

loadErrString takes a `...string`, so you can just `f.loadErrString("Traceback ", "repository name must be canonical")`

also, maybe change the default registry to "bar" (so that it doesn't match the image name) and throw "bar" in the list of expected error text